lib/awful/menu.lua

```diff
@@ -729,6 +764,7 @@ function menu.new(args, parent)
     args = args or {}
     args.layout = args.layout or wibox.layout.flex.vertical
     local _menu = table_update(object(), {
+        select_next = menu.select_next,
```

Comment:
Hm... Why do we want to expose this `select_next` function? (Am I missing something?)